docs/Cloud_integrations/Device_Integration_Center/kafka.md

```diff
@@ -18,17 +18,13 @@ Connect {{ coiote_long_name }} to your **Kafka cluster** or **Azure Event Hub**
 
 To create a new event handler, select: **Integrations** > **Data Integration Center**.
 
-Select **Add event handler**.
+Click **Add event handler** and select **Kafka**.
 
-![Add event handler](images/add-event-handler.png "Add event handler")
-
-Select **Kafka**.
-
-![Add webhook](images/kafka.png "Add webhook")
+![Add event handler](images/add-kafka-handler.webp "Add event handler"){ width=25% }
 
 Provide **basic information** such as the name and description, optionally enabling the option to activate the event handler after creation.  
```

Comment:
Provide **basic information** such as the name and description.
Also, you can activate the event handler after creation.